helpers/apolloClient.ts

```diff
@@ -33,24 +30,18 @@ const whiteList = ['/curriculum']
   }
 })()
 
-function createIsomorphLink() {
-  if (typeof window === 'undefined') {
-    const { SchemaLink } = require('@apollo/client/link/schema')
-    const { schema } = require('../graphql/schema')
-    return new SchemaLink({ schema })
-  } else {
-    const { HttpLink } = require('@apollo/client/link/http')
-    return new HttpLink({
-      uri: '/api/graphql',
-      credentials: 'same-origin'
-    })
-  }
+function createHttpLink() {
+  const { HttpLink } = require('@apollo/client/link/http')
```

Comment:
seems like this doesn't need to be `require`'d dynamically. Could you try importing it

Reply:
I tried importing and in the testfile, it seems that MockFunction was never called [here](https://github.com/garageScript/c0d3-app/blob/master/helpers/apolloClient.test.js#L63)